services/src/test/scala/cromwell/services/metadata/impl/MetadataServiceActorSpec.scala

```diff
@@ -0,0 +1,76 @@
+package cromwell.services.metadata.impl
+
+import java.time.OffsetDateTime
+
+import akka.pattern._
+import com.typesafe.config.ConfigFactory
+import cromwell.core.WorkflowId
+import cromwell.services.ServicesSpec
+import cromwell.services.metadata.MetadataService._
+import cromwell.services.metadata._
+
+class MetadataServiceActorSpec extends ServicesSpec("Metadata") {
+
+  val config = ConfigFactory.load("{}")
```

Comment:
if this is intending to parse that as a String I think you mean `ConfigFactory.parseString`